server/clients/manageRecallsApi/updateRecall.pact.test.ts

```diff
@@ -0,0 +1,103 @@
+// @ts-nocheck
+import { pactWith } from 'jest-pact'
+import { Matchers } from '@pact-foundation/pact'
+import { updateRecall } from './manageRecallsApiClient'
+import * as configModule from '../../config'
+import updateRecallResponseJson from '../../../fake-manage-recalls-api/stubs/__files/update-recall.json'
+
+pactWith({ consumer: 'manage-recalls-ui', provider: 'manage-recalls-api' }, provider => {
+  const accessToken = 'accessToken-1'
+  const recallId = '00000000-0000-0000-0000-000000000000'
+  const recallLength = 'FOURTEEN_DAYS'
+
+  beforeEach(() => {
+    jest.spyOn(configModule, 'manageRecallsApiConfig').mockReturnValue({ url: provider.mockService.baseUrl })
+  })
+
+  describe('update a recall', () => {
+    test('can successfully add recommended recall length to a recall', async () => {
+      await provider.addInteraction({
+        state: 'a recall exists and can be updated',
+        ...updateRecallRequest('an update recall request', recallId, recallLength, accessToken),
+        willRespondWith: updateRecallResponse(Matchers.like(updateRecallResponseJson), 200),
+      })
+
+      const actual = await updateRecall(recallId, recallLength, accessToken)
+
+      expect(actual).toEqual(updateRecallResponseJson)
+    })
+
+    test('returns 400 if blank recall length provided', async () => {
```

Comment:
Should these tests go on the manage-recalls-api side? 
Otherwise every app that's a consumer of manage-recalls-api, will have to write the same PACT tests which are verifying the basic API functionality?
Maybe my still-sketchy understanding of PACT

Reply:
AFAIK no consumer _has_ to verify any condition, e.g. this error behaviour.
OTOH what and when of api side validation is a concern, e.g. should we validate this is positive?
I'd lean towards minimal validation throughout until we need it.